### PR TITLE
Reproduce CORS missing headers

### PR DESCRIPTION
**Pull request by leplatrem**
As a good start, ticket #48 is reproduced in a test case :)

### CODE REVIEW

**Comment by almet** [comment]
Sweet, that will help :)

**Comment by Natim** [comment]
Note that we also probably want forbidden 401 to return JSON.

**Comment by Natim** [comment]
According to https://github.com/spiral-project/daybed/blob/master/daybed/views/errors.py it should be the case.

**Comment by Natim** [comment]
The problem is that if we provide the `Authorization` header that doesn't validates for any reason, pyramid and pyramid_hawk stops the request.

I am not sure about the strategy:
- Should we make sure daybed.js or backbone-daybed doesn't send the Authorization header for OPTIONS request's types.
- Should we allow daybed to answer OPTIONS calls even if the Authorization header is broken?

What do you guys think?

**Comment by almet** [comment]
headers aren't sent in this particular case, but that's not the only problem.

I was working on a patch about that yesterday, but still a work in progress.

**Comment by almet** [comment]
(the missing header is the ORIGIN one, since self.headers isn't sent as part of the request)

**Comment by almet** [comment]
This latest changes should fix it.

Hope the code explains itselfs enough :)

File: daybed/views/errors.py [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
 
 
 def forbidden_view(request):
     if not request.credentials_id or request.credentials_id == Everyone:
-        return Response(
-            '{"error": "401 Unauthorized",'
-            ' "msg": "You must be logged-in to access this page."}',
-            status='401 Unauthorized', content_type='application/json')
-    return Response(
-        '{"error": "403 Forbidden",'
```
Comment (Natim) [inline daybed/views/errors.py:12]: Does HTTPUnauthorized and HTTPForbidden returns a JSON response?

Comment (almet) [inline daybed/views/errors.py:12]: I don't think so, but I don't think that should be a requirement.

the trick I did could also work if we let `Response()` and add the cornice trick just after.

**Comment by almet** [comment]
updated.

**Comment by leplatrem** [comment]
Great! Thanks!

**Pull request merged**